public/update.js

```diff
@@ -0,0 +1,55 @@
+var favicon= new Favico({ // creating a favico instance
+    animation : 'popFade', 
+    bgColor : '#d00',
+    textColor: '#fff', 
+    fontFamily : 'Arial',
+    fontStyle : 'bold',
+
+  });
+
+
+// name of the hidden property and the change event for visibility
+var hidden, visibilityChange; 
+if (typeof document.hidden !== "undefined") { // Opera 12.10 and Firefox 18 and later support 
+  hidden = "hidden";
+  visibilityChange = "visibilitychange";
+} else if (typeof document.mozHidden !== "undefined") {
+  hidden = "mozHidden";
+  visibilityChange = "mozvisibilitychange";
+} else if (typeof document.msHidden !== "undefined") {
+  hidden = "msHidden";
+  visibilityChange = "msvisibilitychange";
+} else if (typeof document.webkitHidden !== "undefined") {
+  hidden = "webkitHidden";
+  visibilityChange = "webkitvisibilitychange";
+}
+ 
+var count =0;
+function handleVisibilityChange() {
+  if (document[hidden]) {
+    //handled in polymer.coffee
+  } else {
+    favicon.reset();
```

Comment:
If there is only an else clause, there is no point in having both if and else. Just keep one.